src/AVFoundation/AVVideoSettings.cs

```diff
@@ -332,23 +335,23 @@ public AVVideoSettingsCompressed (NSDictionary dictionary)
 					throw new ArgumentException ("value");
 				}
 
-				if (v == null)
+				if (v is null)
 					RemoveValue (AVVideo.ScalingModeKey);
 				else
 					SetNativeValue (AVVideo.ScalingModeKey, v);
 			}
 		}
 
-		public AVVideoCodecSettings CodecSettings {
+		public AVVideoCodecSettings? CodecSettings {
 			get {
 				var dict = GetNSDictionary (AVVideo.CompressionPropertiesKey);
-				if (dict == null)
+				if (dict is null)
 					return null;
 				return new AVVideoCodecSettings (dict);
 			}
 
 			set {
-				SetNativeValue (AVVideo.CompressionPropertiesKey, value == null ? null : value.Dictionary);
+				SetNativeValue (AVVideo.CompressionPropertiesKey, value is null ? null : value.Dictionary);
```

Comment:
This is simpler:

```suggestion
				SetNativeValue (AVVideo.CompressionPropertiesKey, value?.Dictionary);
```

There are a few more below as well.